### PR TITLE
Move dconf_settings from vars to defaults

### PR DESCRIPTION
**Pull request by ogajduse**
Ansible does not accept variable specified the way it is shown in the example in Readme. This PR fixes this issue.

### CODE REVIEW

**Comment by ogajduse** [comment]
@jaredhocutt Can you please consider merging this?

File: defaults/main.yml [hunk 1]
```diff
@@ -1,2 +1,4 @@
 ---
-# defaults file for dconf-settings
\ No newline at end of file
+# defaults file for dconf-settings# defaults file for dconf-settings
```
Comment (jaredhocutt) [inline defaults/main.yml]: Looks like you accidentally duplicated this

**Review by jaredhocutt — CHANGES_REQUESTED** [review]
I am so sorry @ogajduse this PR slipped my mind. Yes, I will be happy to merge this. Can you fix the small duplication of the line and I'll get it merged.

Thanks!

File: defaults/main.yml (hunk 1 above)
Comment (ogajduse) [reply defaults/main.yml]: fixed

**Comment by ogajduse** [comment]
Redundancy fixed

**Review by jaredhocutt — APPROVED** [review]
LGTM

**Pull request merged**